### PR TITLE
Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by danarrib**
This is a Brazilian Portuguese translation. 
It's better than the previous submitted ones (#9, #41 and #63) as it:

- Does not include any extra "credits".
- It's perfect 1:1 content, meaning and context match from the original version.
- HTML headers are correctly changed to match pt-BR locale.

Hope you enjoy.

### CODE REVIEW

File: pt-br/index.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  <main>
+    <h1>Não peça para perguntar, apenas pergunte</h1>
+    <p>
+      De vez em quando, nas salas de bate papo online em que eu participo, aparece
+      alguém falando algo parecido com,
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        Alguém que entende de Java por ai?
```
Comment (rffontenelle) [inline pt-br/index.html:25]: ai -> aí

File: pt-br/index.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      </p>
+    </blockquote>
+    <p>
+      Esse é um jeito ruim, por vários motivos. O que a pessoa está
+      <em>realmente</em> perguntando é,
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        Alguém por ai que entende está disposto a se comprometer a olhar o meu
```
Comment (rffontenelle) [inline pt-br/index.html:35]: ai -> aí

File: pt-br/index.html [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        problema, qualquer ele que seja, mesmo se ele não for exatamente relacionado
+        com Java ou mesmo se alguém que não sabe nada de Java puder
+        responder a minha pergunta?
+      </p>
+    </blockquote>
+    <p>
+      Existem várias razões pelas quais as pessoas que TEM o conhecimento
```
Comment (rffontenelle) [inline pt-br/index.html:42]: TEM -> TÊM

File: pt-br/index.html [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+      <p class="message">
+        Alguém por ai que entende está disposto a se comprometer a olhar o meu
+        problema, qualquer ele que seja, mesmo se ele não for exatamente relacionado
+        com Java ou mesmo se alguém que não sabe nada de Java puder
+        responder a minha pergunta?
+      </p>
+    </blockquote>
+    <p>
+      Existem várias razões pelas quais as pessoas que TEM o conhecimento 
+      não admiritiam que tem. E ao perguntar, você está pedindo por mais do que
```
Comment (rffontenelle) [inline pt-br/index.html:43]: admiritiam -> admitiriam

File: pt-br/index.html [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+      pessoas para fora. Eu geralmente respondo perguntas relacionadas a linguagens ou bibliotecas
+      que eu nunca usei, porque as respostas são (de uma forma geral na programação) senso comum.
+    </p>
+    <p>
+      Ou então, isso pode ser visto como..
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+        Eu tenho uma dúvida sobre Java mas eu sou muito preguiçoso para formalizar
```
Comment (rffontenelle) [inline pt-br/index.html:58]: Comma before "mas" (, mas)

File: pt-br/index.html [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+        ela em palavras a menos que haja alguém no canal que esteja disposto a
+        responde-la
```
Comment (rffontenelle) [inline pt-br/index.html:60]: respondê-la

File: pt-br/index.html [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+    </p>
+    <p>
+      A solução é não pedir para perguntar, mas simplesmente perguntar. Alguém que 
+      esteja oscioso no canal e apenas de vez em quando aparece para ver o que está acontecendo
+      dificilmente vai responder ao seu "pedido de pergunta", mas a descrição do seu
+      problema real talvez desperte o interesse e ele dedique algum tempo para responder.
+    </p>
+    <p>
+      Então, para resumir, não pergunte
+      <em>"Alguém que conhece de Java por ai?"</em>,
```
Comment (rffontenelle) [inline pt-br/index.html:75]: ai -> aí

File: pt-br/index.html [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+    <p>
+      Outros problemas similares: <a href="https://xyproblem.info/">O problema XY</a>, <a href="https://nohello.net/">Não diga olá</a>.
+      Leitura aprofundada: <a href="https://stackoverflow.com/help/how-to-ask">Como eu posso fazer uma boa pergunta?</a>,
+      ou se você tem mais tempo: <a href="http://catb.org/~esr/faqs/smart-questions.html">O jeito inteligente de fazer perguntas</a>.
+    </p>
+  </main>
+  <footer>
+    Conteúdo quase totalmente roubado de <a href="https://iki.fi/sol/dontask.html">iki.fi/sol/dontask.html</a>
+    -
+    <a href="https://github.com/maunium/dontasktoask.com">Código fonte no GitHub</a>
```
Comment (rffontenelle) [inline pt-br/index.html:88]: Código-fonte

**Review by rffontenelle — CHANGES_REQUESTED** [review]
Thanks for the PR 🇧🇷. I found some simple typos. Could you please fix them?

**Comment by danarrib** [comment]
> Thanks for the PR 🇧🇷. I found some simple typos. Could you please fix them?

Fixed. Thanks for the heads up!

**Review by rffontenelle — APPROVED** [review]

**Comment by rffontenelle** [comment]
LGTM

**Review by tulir — APPROVED** [review]
Thanks for sorting that out, I'll merge this quickly before there are more duplicates :D

**Pull request merged**